services/types/couchapp/lib/schemas/generic.js

```diff
@@ -3,19 +3,25 @@ exports.v1 = {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "version": "1.0",
   "type": "object",
-  "properties": {
-    "_id": {
+  "description": "this schema provide the overall document structure",
```

Comment:
"Generic document structure" perhaps, so we keep the same wording as for other schemas.